analysis/testing_params.R

```diff
@@ -0,0 +1,250 @@
+    geom_line(alpha=0.3) +
+    facet_wrap(~variable,scales="free_y") +
+    theme_bw()
+
+# Histogram comparing F and H for certain years
+all_gmst_2020 <- filter(all_results,variable=="gmst") %>% filter(year==2020)
+all_gmst_2050 <- filter(all_results,variable=="gmst") %>% filter(year==2050)
+all_gmst_2100 <- filter(all_results,variable=="gmst") %>% filter(year==2100)
+
+all_ohc_2020 <- filter(all_results,variable=="ohc") %>% filter(year==2020)
+all_ohc_2050 <- filter(all_results,variable=="ohc") %>% filter(year==2050)
+all_ohc_2100 <- filter(all_results,variable=="ohc") %>% filter(year==2100)
+
+# GMST histograms
+all_gmst_2020_hist <- ggplot(all_gmst_2020, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Global Mean Surface Temperature Comparison: 2020",x="degC",y="Count") +
+    theme_bw()
+all_gmst_2050_hist <- ggplot(all_gmst_2050, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Global Mean Surface Temperature Comparison: 2050",x="degC",y="Count") +
+    theme_bw()
+all_gmst_2100_hist <- ggplot(all_gmst_2100, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Global Mean Surface Temperature Comparison: 2100",x="degC",y="Count") +
+    theme_bw()
+
+# OHC histograms
+all_ohc_2020_hist <- ggplot(all_ohc_2020, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Ocean Heat Content Comparison: 2020",x="degC",y="Count") +
+    theme_bw()
+all_ohc_2050_hist <- ggplot(all_ohc_2050, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Ocean Heat Content Comparison: 2050",x="degC",y="Count") +
+    theme_bw()
+all_ohc_2100_hist <- ggplot(all_ohc_2100, aes(x=value, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    labs(title = "Ocean Heat Content Comparison: 2100",x="degC",y="Count") +
+    theme_bw()
+
+comparison_hist_grid <- grid.arrange(all_gmst_2020_hist,all_gmst_2050_hist,all_gmst_2100_hist,
+                                     all_ohc_2020_hist,all_ohc_2050_hist,all_ohc_2100_hist,
+                                     nrow = 2)
+
+
+# Load in FaIR parameters for comparison with Hector's
+nc_fairparam <- nc_open("analysis/fair_ar6_climate_params_v4.0.nc")
+fair_param_names <- names(nc_fairparam$var)
+f_ecs <- ncvar_get(nc_fairparam, "F2x")
+nc_close(nc_fairparam)
+
+h_ecs <- params_all$S
+
+# Put the two sets in one dataframe and plot histogram
+h_ecs_df <- data.frame(ecs=h_ecs,model="hector")
+f_ecs_df <- data.frame(ecs=f_ecs,model="fair")
+ecs_all <- bind_rows(h_ecs_df,f_ecs_df)
+
+
+ecs_hist <- ggplot(ecs_all, aes(x=ecs, color=model, fill=model)) +
+    geom_histogram(alpha=0.2, position="identity") +
+    ggtitle("Equilibrium Climate Sensitivity Parameter: Hector vs FaIR") +
+    theme_bw()
```

Comment:
Question? How are you saving the plots to disk? Are you using the Rstudio GUI? or ggsave somewhere?